compiler/record-minmax/driver/Driver.cpp

```diff
@@ -50,6 +50,10 @@ int entry(const int argc, char **argv)
     .type(arser::DataType::FLOAT)
     .help("Record n'th percentile of min");
 
+  arser.add_argument("--parallel_record")
```

Comment:
(optional) Just `--parallel` seems good option since this driver already "record"s min/max.
```suggestion
  arser.add_argument("--parallel")
```